dvc/config_schema.py

```diff
@@ -152,6 +152,8 @@ class RelPath(str):
                     "ssl_verify": Any(Bool, str),
                     "sse": str,
                     "sse_kms_key_id": str,
+                    "sse_customer_algorithm": str,
```

Comment:
Why not reuse `sse` for this? Instead of introducing `sse_customer_algorithm` that is. `sse` also accepts `AES256` I think, and is described as the "SSE encryption algo" to use in docs -- that could be either from KMS or customer-provided.

Reply:
The sole reason of introducing new parameter here was that AWS themselves have kept separate parameters for SSE and SSE-C. Let's say in future AWS brings a scheme where they'll offer to encrypt the data twice. Once using their key and once using users' key. In such case, we will need both parameters. May be a far fetched imagination but not impossible. Also, if the PR gets merged, dvc users can opt not to configure this parameter since the only value AWS now supports is `AES256` which we add by default if `sse_customer_key` is configured. 
It is not like I am of strong opinion to introduce a new parameter here. If majority of reviewers think that it is okay to just reuse the `sse` param, I can make that change as well.

Reply:
> AWS themselves have kept separate parameters for SSE and SSE-C

Makes sense to reflect their config schema then. Thanks for all the context!